Mage.Client/src/main/java/mage/client/dialog/TableWaitingDialog.java

```diff
@@ -450,8 +451,10 @@ private int getPlayersCount(TableView tableView) {
         int playerCount = 0;
         if (tableView != null) {
             for (SeatView seatView : tableView.getSeats()) {
-                if (seatView.getPlayerId() != null && seatView.getPlayerType().equals("Human")) {
-                    playerCount++;
+                if (seatView.getPlayerId() != null) {
+                    if (seatView.getPlayerType().toString().equals("Human")) {
```

Comment:
Oh, fair enough - will make the change :)  That clause of the condition was always returning false as it was previously.